static/js/portico/signup.js

```diff
@@ -4,7 +4,13 @@ $(function () {
 
     $.validator.addMethod('password_strength', function (value) {
         return common.password_quality(value, undefined, $('#id_password, #id_new_password1'));
-    }, "Password is too weak.");
+    }, function () {
+        var password_input = $('#id_password, #id_new_password1');
+        if (password_input.is(':focus')) {
```

Comment:
I'm not sure I quite like this solution. Can it be something like "once you blur the first time, it starts showing, and then never goes away"?